<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GRABAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_persona">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Docente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estudiante</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_tipo_persona">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Persona:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_cedula">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar por Cedula</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_estatura">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Estatura (Cm):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_estatura">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_estatura">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ESTADISTICA DESCRIPTIVA DE LA ESTATURA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_peso">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fecha_de_nacimiento">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Nacimiento:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_peso">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_peso">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ESTADISTICA DESCRIPTIVA DEL PESO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fecha_de_nacimiento">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ESTADISTICA DESCRIPTIVA DE LA FECHA DE NACIMIENTO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_carrera">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_carrera">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_fecha_de_nacimiento">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
